javascript/javascript2/week2/js2-wk2-homework/doubling Numbers/doublingOfNumber.js

```diff
@@ -0,0 +1,6 @@
+"use strict"
+
+let numbers = [1, 2, 3, 4];
+const newNumbers = numbers.filter(OddNumbers => OddNumbers % 2 !== 0).map(multiplyOddNumbers => multiplyOddNumbers * 2);
```

Comment:
variable naming - the number is not odd HERE, so this naming is super confusing. Besides it is ONE number, not many. So simply using `x`, `y` or `num` is actually better here. Same for `multiplyOddNumbers`, it is not an ACTION, it is a NUMBER. Sorry for the caps lock, that was for emphasis.